packages/stream/constants/index.ts

```diff
@@ -0,0 +1,183 @@
+  "Specified stream doesn't exist. Please double check with the sender.";
+export const ERR_NOT_CONNECTED =
+  "There was an issue with the connection - please try to refresh the page.";
+export const ERR_NO_TOKEN_SELECTED = "Please select the token";
+export const ERR_NO_PRIOR_CREDIT =
+  "You don't have enough SOL in your wallet to pay for transaction fees.";
+
+export const PRODUCT_VESTING = "vesting";
+export const PRODUCT_STREAMS = "streams";
+export const PRODUCT_MULTIPAY = "multipay";
+export const PRODUCT_MULTISIG = "multisig";
+
+export const products = [
+  PRODUCT_VESTING,
+  PRODUCT_STREAMS,
+  PRODUCT_MULTIPAY,
+  PRODUCT_MULTISIG,
+];
+
+export const DEFAULT_DECIMAL_PLACES = 2;
+
+export const ERRORS = {
+  amount_required: "Amount is required.",
+  amount_greater_than: "Please provide amount greater than 0.",
+  token_required: "Token is required.",
+  recipient_required: "You must choose a recipient.",
+  not_valid_email: "Must be a valid email.",
+  subject_required: "Please provide a subject (title).",
+  start_date_required: "Start date is required.",
+  start_date_is_in_the_past: "Cannot start stream in the past.",
+  start_time_required: "Start time is required.",
+  start_time_is_in_the_past: "Should start in future.",
+  end_date_required: "End date is required.",
+  end_time_required: "End time is required.",
+  deposited_amount_required: "Deposited amount is required.",
+  amount_too_high: "You don't have enough tokens.",
+  invalid_address: "Please enter a valid Solana wallet address.",
+  address_is_a_program: "Address cannot be a program.",
+  release_amount_greater_than_deposited: "Should be <= deposited amount.",
+  end_should_be_after_start: "End should happen after start.",
+  cliff_should_be_after_start: "Cliff should happen after start.",
+  cliff_should_be_before_end: "Cliff should happen before end.",
+  required: "Required.",
+  release_frequency_is_too_slow:
+    "Should be smaller or equal to difference between END and CLIFF time.",
+  should_be_greater_than_0: "Should be greater than 0.",
+  max_year: "Year should be less than 9999.",
+  subject_too_long:
+    "It is either too long or there are many complex characters.",
+  withdrawal_frequency_too_high:
+    "Withdrawal frequency should be >= release frequency.",
+};
+
+export const PERIOD = {
+  SECOND: 1,
+  MINUTE: 60,
+  HOUR: 3600,
+  DAY: 24 * 3600,
+  WEEK: 7 * 24 * 3600,
+  MONTH: Math.floor(30.4167 * 24 * 3600), //30.4167 days
+  YEAR: 365 * 24 * 3600, // 365 days
+};
+
+export const timePeriodOptions = [
+  { value: PERIOD.SECOND, label: "second" },
+  { value: PERIOD.MINUTE, label: "minute" },
+  { value: PERIOD.HOUR, label: "hour" },
+  { value: PERIOD.DAY, label: "day" },
+  { value: PERIOD.WEEK, label: "week" },
+  { value: PERIOD.MONTH, label: "month" },
+  { value: PERIOD.YEAR, label: "year" },
+];
+
+export const EVENT_CATEGORY = {
+  WALLET: "wallet",
+  STREAM: "stream",
+  VESTING: "vesting",
+};
+
+export const EVENT_ACTION = {
```

Comment:
seems like these were all copied from the app, do we need most of these?

Reply:
Used them for tests only to use our real payloads. Probably going to move them near the tests themselves.